model/event/event_finder.go

```diff
@@ -111,6 +111,14 @@ func RecentSchedulerEvents(distroId string, n int) db.Q {
 	return SchedulerEventsForId(distroId).Sort([]string{"-" + TimestampKey}).Limit(n)
 }
 
+// Admin Events
+// RecentAdminEvents returns the N most recent admin events
+func RecentAdminEvents(n int) db.Q {
+	return db.Query(bson.D{
+		{DataKey + "." + ResourceTypeKey, ResourceTypeAdmin},
+	}).Sort([]string{"-" + TimestampKey}).Limit(n)
```

Comment:
There's no real reason to use a bson.D here, and a bson.M might make more sense?